go/internal/initutil/manager.go

```diff
@@ -123,6 +123,7 @@ type Manager struct {
 			listeners         []grpcutil.Listener
 		} `json:"GRPC,omitempty"`
 	} `json:"Node,omitempty"`
+	InitialisaitonTimeout time.Duration `json:"InitialisaitonTimeout,omitempty"`
```

Comment:
```suggestion
	InitTimeout time.Duration `json:"InitTimeout,omitempty"`
```

(btw, it's `initialization`)